### ISSUE TITLE
 tracklets with features link expired

### ISSUE THREAD

**Issue by iSenya** [Dawn-LX/VidSGG-BIG#18]
Hello! I tried to download the tracklets with features from the author's website http://www.muyadong.com/publication.html
under this paper
Beyond Short-Term Snippet: Video Relation Detection with Spatio-Temporal Global Context, but the link has expired.
Are there any ways to get the data? i have read that you trained your own RCNN to get the tracklet with features for one file that was missing. Should I do the same? Maybe you have the downloaded dataset of features saved somewhere?

**Comment by Dawn-LX** [comment]
refer to the *Data Release Summarize* section in the README.md

here: https://mega.nz/file/UAgnXbjC#YqrlUe2rNl7qEEcr_NbxNk0qPX-Y5O2BZWVhHgqCK9g

and for the missing file, here: https://mega.nz/folder/FMhzFD4A#7R2c98TxgA7Av_b9-_ijYg

**Comment by iSenya** [comment]
Thank you! Got it. If I want to generate my own tracklet files, I'll need to know the settings of Fast-RCNN to generate them, right? You wrote, for example: "we trained our own Faster-RCNN (same training setting as the above paper), and extract the tracklet & features". But I could not find the code for it, the link https://disk.pku.edu.cn/#/link/1CB6184A6D47DAC546E30CC9065151F7 expired.
Do you know how to get the settings?

**Comment by iSenya** [comment]
Is it this one? https://github.com/Dawn-LX/VidVRD-tracklets
?

**Comment by Dawn-LX** [comment]
> Thank you! Got it. If I want to generate my own tracklet files, I'll need to know the settings of Fast-RCNN to generate them, right? You wrote, for example: "we trained our own Faster-RCNN (same training setting as the above paper), and extract the tracklet & features". But I could not find the code for it, the link https://disk.pku.edu.cn/#/link/1CB6184A6D47DAC546E30CC9065151F7 expired. Do you know how to get the settings?

yeah, I also don't have the code from the PKU's author even at the time when their link not expired. they did not provide the training code. 

As for the training setting for this faster-RCNN, I just follow their [paper's](https://pkumyd.github.io/paper/CVPR2020_VideoVRD.pdf) description in Sec 3.1. i.e., the taining data includes trainval set of MS-COCO and the train set of ILSVRC2016-DET, with the same 35 categories in VidVRD. ( In my training, I selected images with at least one category in those 35 categories). For other settings like learning rate and epochs, I just use the default settings in [mask-rcnn benchmark](https://github.com/facebookresearch/maskrcnn-benchmark), or [detectron2](https://github.com/facebookresearch/detectron2).

By the way, In object detection they usually set the training iterations rather than epochs, typically 90k or 180k iterations for COCO trainval set.  So you can estimate the epochs on COCO tranval and convert it to epochs & iterations on your own traing data

as for my training code for training the Faster-RCNN, I have deleted from my server cause our lab had a major reorganize.
